app/models/object_event_hook_config.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# License: AGPL-3.0-or-later WITH WTO-AP-3.0-or-later
+# Full license explanation at https://github.com/houdiniproject/houdini/blob/master/LICENSE
+class ObjectEventHookConfig < ApplicationRecord
+
+  # :webhook_service, #str, webhook service to be called
+  # :configuration, #jsonb, configuration needed to connect to the webhook
+  # :object_event_types, #text (array), a set of object event types
+  # :inbox, #string, inbox for receiving the messages
+
+  belongs_to :nonprofit
+
+  validates :webhook_service, presence: true
+  validates :configuration, presence: true
+  validates :object_event_types, presence: true
+  validates :inbox, presence: true
+
+  WEBHOOK = {
+    open_fn: 'OpenFn'
+  }.freeze
+
+  def webhook
+    case webhook_service
+    when WEBHOOK[:open_fn]
+      Houdini::WebhookAdapter::OpenFn.new(url: inbox, auth_headers: configuration)
+    end
+  end
```

Comment:
I'm not sure about the `webhook_service` attribute name, nor the name for the method to get the corresponding WebhookAdapter. What I imagined here was the listener calling simply for the desired `object_event_hook_config.webhook.post(payload)`.

Reply:
I think `object_event_hook_config.webhook.post(payload)` makes sense. I'm going to make the Webhook Adapters autoloaded from Bess. I can do that myself and i'll create a PR to your branch for review.

Reply:
So I thought about this more. What if it was possible to call something like `object_event_hook_config.fire(payload)` or `object_event_hook_config.transmit(payload)` or something like that? That would wrap the the `new` and `post` call.